lib/src/whisper_manager.py

```diff
@@ -41,6 +43,35 @@ def initialize(self) -> bool:
         try:
             self.temp_dir = self.config.get_temp_directory()
             
+            # Check which backend is configured
+            backend = self.config.get_setting('transcription_backend', 'local')
+            
+            # Configure remote API backend and return early
+            if backend == 'remote':
+                # Validate REST configuration
+                endpoint_url = self.config.get_setting('rest_endpoint_url')
+                
+                if not endpoint_url:
+                    print('ERROR: REST backend selected but rest_endpoint_url not configured')
+                    return False
+                
+                # Validate URL is HTTPS (for security)
```

Comment:
Doesn't match actual function.

We allow both; we could require https which feels better, though extra SSL does add overhead to what are probably local connections.

Maybe just rephrase comment?